test/unit/js/MigrationPersistorTests.js

```diff
@@ -13,43 +13,61 @@ const Errors = require('../../../app/js/Errors')
 // deleteFile: _wrapMethodOnBothPersistors
 // copyFile: copyFileWithFallback
 
-describe('MigrationPersistorTests', function() {
+describe.only('MigrationPersistorTests', function() {
```

Comment:
Let's run all the test suites again :wink:

Reply:
Thanks. `eslint` was supposed to catch this but `.eslintrc` is incorrect. WIll fix this here.